CriticalMass/en.lproj/Localizable.strings

```diff
@@ -133,6 +133,12 @@
 /* No comment provided by engineer. */
 "settings.openSource" = "Github";
 
+/* No comment provided by engineer. */
+"settings.criticalMassDotIn.title" = "Missing a mass?";
+
+/* No comment provided by engineer. */
+"settings.criticalMassDotIn.detail" = "Submit your bike activism to the criticalmass.in project and promote it to other bike enthusiasts.";
```

Comment:
Do you want to add other translations as well or should it be only English?

Reply:
# ⏰ TODO

- [x] Will add more translations 
😄

But I would add them after added to https://crowdin.com/project/critical-maps